common/internationalization/lib/src/app_internationalization.dart

```diff
@@ -153,8 +154,28 @@ class AppInternationalization extends Translations {
           'pendingTransfer': 'Pending transfers',
           'yesterday': 'Yesterday',
           'sentRequestBySmsTitle': 'Do you want to send your request by sms?',
+          'connectionRestore': 'Back online',
+          'emptyForfeit': 'No forfeit found',
```

Comment:
What  is the use case of this `emptyForfeit` key?
Why not only use the `noForfeitFound` key since they have the same translation.